gpt4all-api/makefile

```diff
@@ -1,6 +1,6 @@
 ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 APP_NAME:=gpt4all_api
-PYTHON:=python3.8
+PYTHON:=python3.11
```

Comment:
Why? I'm not really into Python but bumping versions randomly is usually a bad idea

Reply:
Hi, 
The `test_build`  command in this `makefile` uses the  [gpt4all_api/Dockerfile.buildkit](https://github.com/nomic-ai/gpt4all/blob/36f7fb584824961dc692c9f2354ee8f60c50587b/gpt4all-api/gpt4all_api/Dockerfile.buildkit#L2) to build the image. that Dockerfile uses the `tiangolo/uvicorn-gunicorn:python3.11` as its base image.
about the versions bumping - in my opinion, the best is to wrap the code with tests to make sure that nothing breaks on the version upgrade  and to add the testing to the CI pipeline